### DISCUSSION TITLE
qtile with python can gui app ?

### DISCUSSION THREAD

**Discussion by ghost** [qtile/qtile#3913] [Ideas]
Is it possible to develop interface library using qtile system like Gnome?

Will qtile developers write display manager in the future?
https://github.com/fairyglade/ly

**Comment by elParaguayo** [comment]
Not sure I understand the first question. What you mean by "interface library"?

As for the second question: no, we're not planning on writing a display manager. Any reason for asking?

**Reply by ghost** [reply]
1 - like python gui library but using qtile library
2 - ly can't support color and sometimes qtile + ly be slow

**Reply by elParaguayo** [reply]
1 - Qtile is a window manager. It's not really designed for creating guis.

You can use the Popup Toolkit in qtile-extras (https://qtile-extras.readthedocs.io/en/stable/manual/how_to/popup.html) but it will never compare to more developed libraries

2 - there are plenty of other display managers around if you don't like ly

**Reply by ghost** [reply]
why qtile developer not make more :( okay

**Reply by ghost** [reply]
ı love you guys but :(

**Reply by elParaguayo** [reply]
Sorry to upset you but neither of your requests are functionality that a window manager should be providing.

**Reply by ghost** [reply]
:( okay sorry
